guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -211,6 +211,36 @@ To configure network services on {Project}'s integrated {SmartProxy}, complete t
 # hammer proxy info --name "_{foreman-example-com}_"
 ----
 
+[[Configuring_Networking-Multiple_DHCP_Subnets_via_Puppet]]
+==== Multiple DHCP Subnets via Puppet
+
+The installer have options available only for a single DHCP subnet. One way to define more than one subnet is via Puppet, the tool that the installer utilizes to perform the configuration.
+
+For every additional subnet, create an entry in `/etc/foreman-installer/custom-hiera.yaml` file:
+
+----
+dhcp::pools:
+ example.com:
+   network: 192.168.99.0
+   mask: 255.255.255.0
+   gateway: 192.168.99.1
+   range: 192.168.99.5 192.168.99.49
+----
+
+Execute {foreman-installer} to perform the changes and verify that the `/etc/dhcp/dhcpd.conf` contains appropriate entries. Subnets must be then defined in {Project} database.
+
+[[Configuring_Networking-Multiple_DHCP_Subnets_Manually]]
+==== Multiple DHCP Subnets Manually
```

Comment:
We should discourage this and mention that you are on your own to ensure things work. For example, users who upgrade won't get HTTPBoot support working out of the box. There are no instructions for users to maintain this.

Reply:
I disagree. Then remove the option from the installer.

Reply:
What I'm saying is that the user must be able to understand the implication. It's a trade off: they get flexibility, but also no support in upgrades. If a fix or feature in the DHCP config is needed, they are responsible. HTTPBoot is one example.

Reply:
My point is I am getting asked this quite often - I am not using Puppet, I have/want to deploy my own DNS server. And I would like to have a place where I can just share link to.

I can just hide this downstream and only show this upstream if that makes it any better.

Reply:
My point is that this option is valid, but users should understand the implication. Right now this looks like a thing without downsides.

Imagine a user did this with 6.7 and is happy. Then 6.8 comes around and HTTPBoot doesn't work. So support is contacted, it takes a while to figure out and we will end up with various suggestions on how to configure HTTPBoot. There will be frustration.

That's why the user must be informed about the trade off. If they manually manage their server, it's their responsibility and certain features may not work unless they configure it the right way.

Reply:
I have deleted the whole block.